docs/rackhd/install_os.rst

```diff
@@ -201,6 +201,26 @@ Add following block into httpProxies in /opt/monorail/config.json
     #end of file
     ###################
 
+**Debian**
+For **live** installation, see this payload https://github.com/RackHD/RackHD/blob/master/example/samples/install_debian_payload_minimal.json Remember to replace **repo** with your own **{fileServerAddress}:{fileServerPort}/debian**, you can find the proper parameters in /opt/monorail/config.json
+
+Add following block into httpProxies in /opt/monorail/config.json
+
+  .. code::
+
+    {
+      "localPath": "/debian",
+      "server": "http://ftp.us.debian.org/",
+      "remotePath": "/debian/"
+    }
+
+  .. code::
+
+    # Create workflow
+    # Replace the 9090 port if you are using other ports
+    # You can configure the port in /opt/monorail/config.json -> 'httpEndPoints' -> 'northbound-api-router'
+    curl -X POST -H 'Content-Type: application/json' -d @install_debian_payload_minimal.json 127.0.0.1:9090/api/current/nodes/{node-id}/workflows?name=Graph.InstallDebian | jq '.'
```

Comment:
These paragraphs to **live** install Debian is similar to that in Ubuntu. They shouldn't be under the `Making the Mirrors` section. @mcgG will refine them later.